Core/TimberApi/LocalizationSystem/LocalizationFetcher.cs

```diff
@@ -27,7 +27,7 @@ internal static class LocalizationFetcher
                 string id = defaultRecord.Id;
                 if (!localizedRecords.TryGetValue(id, out string text) || string.IsNullOrEmpty(text))
                 {
-                    localizedRecords[id] = defaultRecord.Text;
+                    localizedRecords[id] = TextColors.ColorizeText(defaultRecord.Text);
```

Comment:
This is fine, but won't fix the issue you having since it's for the fallback.

Change the localizedRecords line to
`            Dictionary<string, string> localizedRecords = GetLocalizationRecordsFromFiles(localizationKey, GetLocalizationFilePathsFromDependencies(localizationKey))
                .ToDictionary(record => record.Id, record => TextColors.ColorizeText(record.Text));`

Then it works for all

Reply:
Done